src/Sarif/Baseline/V2/V2ResultMatcher.cs

```diff
@@ -29,16 +29,19 @@ public IList<MatchedResults> Match(IList<ExtractedResult> before, IList<Extracte
     /// </remarks>
     internal class StatefulResultMatcher
     {
+        // Threshold for how many 'nearby' Results are considered after other match phases
+        private const int NearnessThreshold = 3;
+
         // Results in the first set.
         private List<ExtractedResult> Before { get; }
+        private TrustMap BeforeTrustMap { get; }
```

Comment:
>BeforeTrustMap [](start = 25, length = 14)

Why are these properties and not fields?

Reply:
I use fields only when the performance of the getter method might be an issue. I tend to prefer properties when possible.

Reply:
This is very inconsistent with the rest of the code base. I think it alters the readability of the code. How strongly do you feel about retaining this for your repo contributions? If it's important, we can be consistent in this pattern for the Baseline API. I'd need to be convinced it's useful before blessing it as a universal standard. It compromises code readability from my perspective. 

---
In reply to: [390520014](https://github.com/microsoft/sarif-sdk/pull/1807#discussion_r390520014) [](ancestors = 390520014)

Reply:
I don't disagree with this one. The code is mostly consistent. I've changed all of my private property usage to fields with underscore prefix.

I think other "style above naming" would be better discussed in threads, settled, and then fixed up across the codebase.